pages/linux/zypper.md

```diff
@@ -21,3 +22,7 @@
 - Search package via keyword:
 
 `zypper search {{keyword}}`
+
+- Show a list with information related with configured repositories:
```

Comment:
```suggestion
- Show information related to configured repositories:
```
Would this also be correct? (`list of information` = `information` in my opinion)

Reply:
yep! Maybe better in that way!